ecs_logging/_stdlib.py

```diff
@@ -112,6 +112,24 @@ def __init__(self, stack_trace_limit=None, exclude_fields=()):
         self._exclude_fields = frozenset(exclude_fields)
         self._stack_trace_limit = stack_trace_limit
 
+    def _record_error_type(self, record):
```

Comment:
Would be good to have an explict `return None` at the bottom of each of these functions

Reply:
done